### PR TITLE
RIP forceSpellCheck (whatwg/html#2904)

### PR DESCRIPTION
**Pull request by ayg**

### CODE REVIEW

**Comment by ayg** [comment]
whatwg/dom#2904

Why no autolinking in the title?

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None



View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/17321)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263894334)

**Comment by domenic** [comment]
It never made it into any interfaces IDL files?

File: interfaces/html.idl [hunk 1]
```diff
@@ -137,7 +137,6 @@ interface HTMLElement : Element {
   readonly attribute DOMString accessKeyLabel;
   [CEReactions] attribute boolean draggable;
   [CEReactions] attribute boolean spellcheck;
-  void forceSpellCheck();
```
Comment (ayg) [inline interfaces/html.idl]: ?

Comment (domenic) [inline interfaces/html.idl]: Sorry, somehow didn't see this.

**Review by domenic — APPROVED** [review]

**Comment by ayg** [comment]
Superseded by #6858, so I'll close for now.  Reopen if that one isn't accepted right now for some reason.

**Pull request closed without merging**